nil pointer dereference when a plugin doesn't exist

If there's a plugin listed in `verless.yml` that doesn't exist, verless will not only print a warning, but also panic with the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0xb5fe82]

goroutine 7 [running]:
github.com/verless/verless/core.(*Build).Run(0x0, 0xc00013c010, 0xc86b4f)
        C:/projects/verless/core/build.go:161 +0xb2
github.com/verless/verless/core.Serve.func1(0xc000018b40, 0xc00028a4e0, 0xe599a0, 0xc00028f020, 0xc0001100a0, 0x7, 0xc000347540, 0xc00028a480)
        C:/projects/verless/core/serve.go:87 +0x1f6
created by github.com/verless/verless/core.Serve
        C:/projects/verless/core/serve.go:70 +0x241
exit status 2
```

This should be fixed.

--- a/core/serve.go
+++ b/core/serve.go
@@ -74,6 +74,13 @@ func Serve(path string, options ServeOptions) error {
 		for {
 			select {
 			case _, ok := <-rebuildCh:
+				finishFirst := func() {
+					if isFirst {
+						initialBuild.Done()
+						isFirst = false
+					}
+				}
+
 				if !ok {
 					return
 				}
@@ -82,18 +89,19 @@ func Serve(path string, options ServeOptions) error {
 				build, err := NewBuild(memMapFs, path, options.BuildOptions)
 				if err != nil {
 					out.Err(style.Exclamation, "failed to initialize new build: %s", err.Error())
+					finishFirst()
+					continue
 				}
 
 				if err := build.Run(); err != nil {
 					out.Err(style.Exclamation, "failed to build the project: %s", err.Error())
+					finishFirst()
+					continue
 				}
 
 				out.T(style.HeavyCheckMark, "project built successfully")
 
-				if isFirst {
-					initialBuild.Done()
-					isFirst = false
-				}
+				finishFirst()
 			case _, ok := <-done:
 				// Stops the goroutine if requested to.
 				// Triggers on closing of the done channel.

